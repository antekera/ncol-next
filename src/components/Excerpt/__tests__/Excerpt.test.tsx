import React from 'react'

import { render } from '@testing-library/react'

import { Excerpt } from '..'

describe('Excerpt', () => {
  test('should be defined', () => {
    const { container } = render(<Excerpt />)
    expect(container.firstChild).toBeDefined()
  })
})
