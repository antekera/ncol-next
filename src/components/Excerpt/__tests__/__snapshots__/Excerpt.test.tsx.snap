// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Excerpt should match snapshots 1`] = `
<p
  class="text-sm text-slate-500 sm:text-base lg:text-base dark:text-neutral-300"
>
  Cómo reservado y delicado fue calificado el estado de salud del Alcalde del Municipio Miranda Jorge Nava, luego de ser víctima de un sicariato en horas de la madrugada de este martes en esta localidad. Nava, se encontraba participando en la conocida Feria conocida como vuelta al terruño en el sector el Terraplén de esta 

   ...
</p>
`;
